### PR TITLE
acrn-common: enable UIO and UIO_PCI_GENERIC configs

### PR DESCRIPTION
**Pull request by liuyuan-GitHub**
Enable UIO and UIO_PCI_GENERIC guest kernel configurations
for acrn shared memory virtual device

Signed-off-by: Yuan Liu <yuan1.liu@intel.com>

### CODE REVIEW

**Comment by wenlingz** [comment]
@saininav please help review and merge, thanks

File: recipes-kernel/linux/files/acrn-common.cfg [hunk 1]
```diff
@@ -1 +1,3 @@
 CONFIG_ACRN_VIRTIO_DEVICES=y
+CONFIG_UIO=m
+CONFIG_UIO_PCI_GENERIC=m
```
Comment (saininav) [inline recipes-kernel/linux/files/acrn-common.cfg:3]: Are these configurations applicable to both SOS and UOS ?
Are these configurations applicable to both 4.19 and 5.4 kernels ?

**Comment by cheeyanglee** [comment]
please consider to use uio.scc from yocto-kernel-cache for available configuration.
see http://git.yoctoproject.org/cgit/cgit.cgi/yocto-kernel-cache/tree/features/uio/uio.scc?h=yocto-5.4

File: recipes-kernel/linux/files/acrn-common.cfg (hunk 1 above)
Comment (liuyuan-GitHub) [reply recipes-kernel/linux/files/acrn-common.cfg:3]: Only for UOS, for both of 4.19 and 5.4 kernels

Comment (wenlingz) [reply recipes-kernel/linux/files/acrn-common.cfg:3]: @saininav  please give some advice about this PR, which is just for UOS. I found that sos also use acrn-common.cfg, so this PR  need modify. Could you please help generate new PR for this, both on master-next and dunfell ? just close 122 and 123, thanks

**Comment by saininav** [comment]
CONFIG_UIO=m is already available with common configuration.

https://github.com/intel/meta-acrn/commit/9faafd8061476da9427a7e276b23c283f57398f7

**Pull request closed without merging**